ci/test/recursive

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+source ./common
+init_test_repo
+cd $test_repo_path
+
+function create_workflow(){
+    cat > $1 << EOF
+workflow "test" {
+  resolves = "show env"
+}
+
+action "show env" {
+  uses = "actions/bin/sh@master"
+  args = "ls"
+}
+EOF
+}
+
+mkdir another
+mkdir another/another
+
+workflows=( "main.workflow" "another/main.workflow" "another/another/main.workflow" "try.workflow" "another/demo.workflow" )
+
+for i in "${workflows[@]}"
+do
+    create_workflow $i
+done
+
+popper run --recursive
```

Comment:
we can check here whether each of them got invoked. Maybe by saving output to a file and then running `grep` on that file to search for output that indicates that all workflows were executed.

Reply:
had to unset `set -x`  in this [line](https://github.com/systemslab/popper/blob/158160ffb54ea0e84f08a225c85096443403f694/ci/test/recursive#L36). `set -x` wasn't able to follow through nested for loops and proper output wasn't displayed.